crates/forge_loader/src/manifest.rs

```diff
@@ -84,28 +242,110 @@ pub struct ForgeModules<'a> {
+    #[serde(rename = "compass:teamPage", default, borrow)]
+    team_page: Vec<CommonKey<'a>>,
+    // confluence Modules
+    #[serde(rename = "confluence:contentAction", default, borrow)]
+    content_action: Vec<CommonKey<'a>>,
+    #[serde(rename = "confluence:contentByLineItem", default, borrow)]
+    content_by_line_item: Vec<ContentByLineItem<'a>>,
+    #[serde(rename = "confluence:contextMenu", default, borrow)]
+    context_menu: Vec<CommonKey<'a>>,
+    #[serde(rename = "confluence:globalPage", default, borrow)]
+    confluence_global_page: Vec<CommonKey<'a>>,
+    #[serde(rename = "confluence:homepageFeed", default, borrow)]
+    homepage_feed: Vec<CommonKey<'a>>,
+    #[serde(rename = "confluence:spacePage", default, borrow)]
+    space_page: Vec<CommonKey<'a>>,
+    #[serde(rename = "confluence:spaceSettings", default, borrow)]
+    space_settings: Vec<CommonKey<'a>>,
+    #[serde(rename = "macro", default, borrow)]
+    macros: Vec<MacroMod<'a>>,
+    // jira modules
+    #[serde(rename = "jira:adminPage", default, borrow)]
+    pub jira_admin_page: Vec<JiraAdminPage<'a>>,
+    #[serde(rename = "jira:customField", default, borrow)]
+    pub custom_field: Vec<CustomField<'a>>,
+    #[serde(rename = "jira:customFieldType", default, borrow)]
+    custom_field_type: Vec<CustomFieldType<'a>>,
+    #[serde(rename = "jira:dashboardBackgroundScript", default, borrow)]
+    dashboard_background_script: Vec<CommonKey<'a>>,
+    #[serde(rename = "jira:dashboardGadget", default, borrow)]
+    dashboard_gadget: Vec<DashboardGadget<'a>>,
+    #[serde(rename = "jira:globalPage", default, borrow)]
+    jira_global_page: Vec<CommonKey<'a>>,
+    #[serde(rename = "jira:issueAction", default, borrow)]
+    issue_action: Vec<CommonKey<'a>>,
+    #[serde(rename = "jira:issueContext", default, borrow)]
+    issue_context: Vec<IssueClass<'a>>,
+    #[serde(rename = "jira:issueGlance", default, borrow)]
+    issue_glance: Vec<IssueClass<'a>>,
+    #[serde(rename = "jira:issuePanel", default, borrow)]
+    issue_panel: Vec<CommonKey<'a>>,
+    #[serde(rename = "jira:issueViewBackgroundScript", default, borrow)]
+    issue_view_background_script: Vec<CommonKey<'a>>,
+    #[serde(rename = "jira:jqlFunction", default, borrow)]
+    jql_function: Vec<CommonKey<'a>>,
+    #[serde(rename = "jira:projectPage", default, borrow)]
+    project_page: Vec<CommonKey<'a>>,
+    #[serde(rename = "jira:projectSettingsPage", default, borrow)]
+    project_settings_page: Vec<CommonKey<'a>>,
+    #[serde(rename = "jira:uiModificatons", default, borrow)]
+    pub ui_modifications: Vec<UiModificatons<'a>>,
+    #[serde(rename = "jira:workflowValidator", default, borrow)]
+    pub workflow_validator: Vec<WorkflowValidator<'a>>,
     #[serde(rename = "jira:workflowPostFunction", default, borrow)]
-    workflow_post_functions: Vec<WorkflowPostFunction<'a>>,
+    pub workflow_post_function: Vec<WorkflowPostFunction<'a>>,
+    // Jira Service Management Modules
```

Comment:
> Asides from the requested changes, could you also add the invokable [Jira Service Management](https://developer.atlassian.com/platform/forge/manifest-reference/modules/index-jsm/) and [https://developer.atlassian.com/platform/forge/manifest-reference/modules/index-bitbucket/](https://github.com/atlassian-labs/FSRT/pull/Bitbucket) modules as well?

Confusion moment here. So I added all the JSM Modules because they all have a function property and resolver property that can take a string referencing a function. Some might be as simple as returning a component but nonetheless. I'm a bit confused here, because I saw your later comment that said the modules looked comprehensive when JSM modules weren't included yet. 

Just wanted to make a note of it here. And clear this up. yee.

Reply:
Ah I missed those, yeah those should be added too.